snap/info.go

```diff
@@ -1305,6 +1305,13 @@ func ReadInfoFromSnapFile(snapf Container, si *SideInfo) (*Info, error) {
 		return nil, err
 	}
 
+	// As part of the validation, also read the snapshot manifest file: we
+	// don't care about its contents now, but we need to make sure it's valid.
+	_, err = ReadSnapshotYamlFromSnapFile(snapf)
```

Comment:
(nitpick^3) this could be slightly more compact in a single line: `if _, err := Read...; err != nil {`

Reply:
Yes! :-) That was my initial approach, then I noticed the code above, and chose to keep the same style. I'm happy to change as you suggested, if no one else complains. :-)

Reply:
It's such a detail, don't worry. FWIW, I like the compact style better and (afaict) most of our code follows it when appropriate but we are not super stringent here so it's fine :)